fastlane/metadata/android/es-ES/short_description.txt

```diff
@@ -1 +1 @@
-Configuración individual de volumen de voz y música para dispositivos Bluetooth.
\ No newline at end of file
+Configuración individual del volumen de la voz y la música para los dispositivos Bluetooth.
```

Comment:
The text can only be 80 characters long, is it possible to make it shorter?

Reply:
Configuración separada del volumen de voz y música para dispositivos Bluetooth

Configura el volumen individual de voz y música para dispositivos Bluetooth

Administra el volumen para la música y las llamadas en dispositivos Bluetooth.

Reply:
I don't understand Spanish, I trust your judgement, select something you like :slightly_smiling_face:.

Reply:
Good afternoon @d4rken

Sorry for not telling you which one I would put.

I would put:

Administra el volumen para la música y las llamadas en dispositivos Bluetooth.

Sorry for the mistake of not telling you which one fits best. I don't know where I left my head...


I apologize again, it seems that my neuron went on a weekend or a party.


A hug